apps/designer/components/TemplateCards/TemplateCard.ts

```diff
@@ -0,0 +1,50 @@
+import { onBoardUiStore as boardStore } from "../../globalStore/Navigation-Store/templateCardStore/onBoardUiStore";
+const { openEuropaCard, openJayantiCard, openYamunaCard } = boardStore();
+const cards: Array<{
+  name: string;
+  id: number;
+  size: string;
+  price: string;
+  myfunc: () => void;
+  styleWhenEuropaCardOpen: string;
```

Comment:
why do you need three different components? @NamanRajputXXI 

This implementation is very clunky. All that should be needed is one card component

Reply:
I made 3 different component but i removed them these are some extra files in code .